js/src/redux/providers/registry/middleware.js

```diff
@@ -91,6 +131,17 @@ export default (api) => (store) => {
           clearTimeout(timeout);
         }
 
+        write.flush();
+
+        break;
+      case 'setReverse':
+        write(
+          () => store.getState().nodeStatus.netChain,
```

Comment:
Why don't we just pass the values in as opposed to passing in a function that retrieves the values?